src/client/testing/common/types.ts

```diff
@@ -90,6 +93,12 @@ export enum TestStatus {
     Skipped = 'Skipped',
     Pass = 'Pass',
 }
+export type FinalTestStatus = TestStatus.Fail | TestStatus.Error | TestStatus.Skipped | TestStatus.Pass;
+export type NonPassingTestStatus = Exclude<FinalTestStatus, TestStatus.Pass>;
+
+export function isNonPassingTestStatus(status: TestStatus): boolean {
+    return [TestStatus.Fail, TestStatus.Error, TestStatus.Skipped].includes(status);
```

Comment:
I would recommend using logic similar to what you used to build the `NonPassingTestStatus` union type, instead of explicitly listing all test statuses.

```suggestion
    const nonPassing = Object.values(TestStatus).filter((value) => value !== TestStatus.Pass);
    return nonPassing.includes(status);
```

Reply:
As I said in the note above `TestStatus`, I found it a little too tricky to bother with right now.

Reply:
Which part is tricky? This suggestion follows the same logic as how `NonPassingTestStatus` is defined, by excluding the same value as `NonPassingTestStatus`, instead of including all non-passing statuses one by one.

Reply:
Ah, sorry, I misunderstood.  Your suggestion is a good one.